### PR TITLE
Fix & test for #988: column_filters in not compatible with babel.

### PR DESCRIPTION
**Pull request by torotil**
Filters currently have two issues with translatable column labels:

 1. _LazyString is not hashable and can't be used as a dict key.
 2. It can't be JSON serialized.

This patch creates an explicit FieldGroup class to store the label of the FieldGroup and it's filters. Now we can use the untranslated FieldGroup label as dict-key while not losing the translatability of the label.

### CODE REVIEW

File: flask_admin/model/base.py [hunk 1]
```diff
@@ -61,6 +61,26 @@ def clone(self, **kwargs):
         return ViewArgs(**kwargs)
 
 
+class FilterGroup:
```
Comment (mrjoes) [inline flask_admin/model/base.py]: New style class please.

**Comment by jmagnusson** [comment]
Thanks for this fix @torotil. Did it for me at least.

@mrjoes Could we have this merged?

**Comment by mrjoes** [comment]
I'd love to, but it breaks the build.

**Comment by torotil** [comment]
That's due to the changes in flask-admin:master that happened in the meanwhile. Looking into it.

**Comment by torotil** [comment]
Now it builds.

**Comment by jmagnusson** [comment]
Nice @mrjoes. Could we have a new PyPI release with this?

**Pull request merged**